rpc/src/rpc.rs

```diff
@@ -552,6 +590,11 @@ impl JsonRpcRequestProcessor {
             }
         }
 
+        let bank = self.get_bank_with_config(slot_context)?;
+        let partitioned_epoch_reward_enabled = bank
+            .feature_set
+            .is_active(&feature_set::enable_partitioned_epoch_reward::id());
```

Comment:
I think I need to make this logic more complex; ie. check the `activated_at` slot. We need to know if the feature was active during the historical epoch, not just currently.

Reply:
`FeatureSet::activated_slot()` ftw

Reply:
Or we can check whether the reward partition data account exists?

Reply:
I used `activated_slot()`